data-reconciliation-app/src/app.json

```diff
@@ -1,21 +1,53 @@
 {
   "endpoints": {
-    "/votes": {
-      "get": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "read",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
-        "mode": "readonly",
-        "openapi": {}
-      },
+    "/ingest": {
       "post": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "write",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
+        "js_module": "endpoints/ingest.js",
+        "js_function": "submitData",
+        "forwarding_required": "always",
+        "authn_policies": ["member_cert"],
```

Comment:
Should this be user_cert?

Reply:
I've modified it to accept calls from both members and users; later, I will change the test cases to test both scenarios.